installer/configurator/configurator.go

```diff
@@ -69,35 +110,51 @@ func main() {
 		log.SetLevel(log.InfoLevel)
 	}
 
-	log.Info("Starting installer version ", InstallerVersion)
+	sender, err := NewTelemetrySender(*agentUri, "rum-installer-nginx", "prod")
+	if err != nil {
+		handleError(err, sender, "", *dryRun)
+	}
 
 	if *dryRun {
 		log.Info("Dry run enabled. No changes will be made.")
 	}
 
 	if err := validateInput(*appID, *site, *clientToken, *arch, *sessionSampleRate, *sessionReplaySampleRate); err != nil {
-		handleError(err)
+		handleError(err, sender, *appID, *dryRun)
 	}
 
 	var configurator ProxyConfigurator = &NginxConfigurator{}
 
 	if err := configurator.VerifyRequirements(); err != nil {
-		handleError(err)
+		handleError(err, sender, *appID, *dryRun)
 	}
 
 	if err := configurator.DownloadAndInstallModule(*arch, *skipVerify); err != nil {
-		handleError(err)
+		handleError(err, sender, *appID, *dryRun)
 	}
 
 	if err := configurator.ModifyConfig(*appID, *site, *clientToken, *agentUri, *sessionSampleRate, *sessionReplaySampleRate, *dryRun); err != nil {
-		handleError(err)
+		handleError(err, sender, *appID, *dryRun)
 	}
 
 	if !*dryRun {
 		if err := configurator.ValidateConfig(); err != nil {
-			handleError(err)
+			handleError(err, sender, *appID, *dryRun)
 		}
 
 		log.Info("Datadog NGINX module has been successfully installed and configured. Please reload NGINX or restart the service for the changes to take effect")
 	}
+
+	if !*dryRun {
+		log.Debug("Sending installation telemetry")
+		if err := sender.SendInstallationCount("success", ""); err != nil {
+			log.Error("Error sending installation success count: ", err)
+		}
+
+		end := time.Now().UnixNano() / int64(time.Millisecond)
```

Comment:
[nit] Probably don't make a huge difference, however, I would argue to set `end` right after `if !*dryRun` for accuracy